packages/exec/__tests__/exec.test.ts

```diff
@@ -286,6 +286,27 @@ describe('@actions/exec', () => {
     expect(stderrCalled).toBeTruthy()
   })
 
+  it('Handles stdin', async () => {
```

Comment:
This test isn't currently working on windows. You may want to create a 
wait-for-input.cmd
```
@echo off
set /p input=
echo %input%
``` 
and invoke that if `IS_WINDOWS` is true. We do a similar thing here: https://github.com/actions/toolkit/blob/master/packages/exec/__tests__/exec.test.ts#L453

For example
```
    let command: string
    if (IS_WINDOWS) {
      command = './wait-for-input.cmd'
    } else {
      command = './wait-for-input.sh'
    }
    const waitForInput: string = path.join(
      __dirname,
      'scripts',
      command
    )
```

Reply:
Done